app/controllers/applications_controller.rb

```diff
@@ -107,6 +107,23 @@ def show
     # Required for new email alert signup form
     @alert = Alert.new(address: @application.address)
 
+    if current_user && current_user.admin?
+      if params[:with_councillors] == "true"
+        @application_authority_councillors = [{ id: "1", name: "Mark Gardiner", party: "Independent" },
```

Comment:
Addressed in https://github.com/openaustralia/planningalerts/commit/3deae0c749e1243e7b78cc9cae867cb44d257450